Automattic/newspack-plugin
# Update the style of the Modal component

### All Submissions:






### Changes proposed in this Pull Request:

I noticed the `Modal` felt slightly off, especially with the header and the various paddings. This PR restyle it to use a 64px padding around it. It also moves the title more inline with the content. For buttons, we are now using a `buttonsCard`.

I had to reorganise things in the Campaigns to make sure the changes were not breaking the "Add New Prompt" modal. And as an added bonus, I've tweaked the "Add New Prompt" modal to something similar to what was suggested in #861 

### How to test the changes in this Pull Request:

1. Check the various Modals (Components Demo, Campaigns, Analytics -- Custom Events, etc...)
2. Switch to this branch
3. Test again

### Other information:
